modules/tensor_mechanics/doc/content/source/materials/ADRankTwoCylindricalComponent.md

```diff
@@ -0,0 +1,27 @@
+# Rank Two Cylindrical Component
+
+!syntax description /Materials/ADRankTwoCylindricalComponent
+
+## Description
+
+This is an ADMaterial model used to extract components of a rank-2 tensor in a
+cylindrical coordinate system. This can be used regardless of the coordinate
+system used by the model.
+
+This ADMaterial model is used by
```

Comment:
Make the same minor tweaks (is used by, called directly, etc) that I suggested for the model above.